points/module.py

```diff
@@ -282,6 +166,55 @@ def _get_board(
             result.append(template.format(points=db_user.points, name=name))
         return "\n".join(result)
 
+    @staticmethod
+    def _create_embeds(
+        ctx,
+        title: str,
+        description: str,
+        order: BoardOrder,
+        element_count: int,
+        page_count: int,
+    ) -> list[discord.Embed]:
```

Comment:
This needs to be `List[discord.Embed]`. Python 3.8 does not support `list[]`, we have to use the `typing` version of it.